modules/ROOT/pages/basketball-robot-guide/mechanics/cam.adoc

```diff
@@ -54,6 +82,32 @@ image::fusion_360_cam_stock_settings.png[]
 
 Now you can click OK, and you have created your first setup for milling job.
 
+== Order of operations
+
+Generally move from inside the part to outside.
+Cutting from outside first can mean that the part comes loose and the inside features can't be cut.
+
+Example order:
+
+. Drilling
+. Pockets
+. Inside contours
+. Outside contours
+
+== Tool size
+
+Largest available diameter tool size should be used based on smallest inner radius of the milled parts.
+Larger diameter tools are more rigid and can remove more material in the same time.
+
+== Tools
+
+The file for the tools you can use can be found below at <<_tool_settings>>.
+
+Import these into your local tool library, by opening up MANAGE -> Tool Library. +
```

Comment:
Instead of using plus at the end of the line to have each sentence on a separate line, it would be better to have these sentences in a list.